poms/bom/pom.xml

```diff
@@ -5999,28 +5999,28 @@
                 <version>${jsoup.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.kie.soup</groupId>
-                <artifactId>kie-soup-commons</artifactId>
-                <version>${optaplanner.version}</version>
+                 <groupId>org.mvel</groupId>
```

Comment:
Nitpick: Isn't there one space too much here and on other changed lines of this file?

Reply:
@ppalaga : It should be better right now.